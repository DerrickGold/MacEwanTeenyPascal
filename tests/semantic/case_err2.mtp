(*Try case with string literal*)
const test := 'error';
var a, b : integer;


begin

	case a + b of
	'error2':
		a := b
	;
	test:
		b := a
	end


end.