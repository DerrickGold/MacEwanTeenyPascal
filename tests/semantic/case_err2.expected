
Semantic Error [line 9]: value: "literal0"
	Unexpected l-value type: 
Expected type(s) 'Integer' or 'Boolean'; found 'String' instead.
